docs/docsite/rst/porting_guides/porting_guide_2.6.rst

```diff
@@ -62,7 +62,7 @@ Noteworthy module changes
   destination path like this::
 
     $ ansible localhost -m file -a 'path=/tmp/lib state=directory'
-
+* Openstack modules are no longer using ``shade`` library. Instead ``openstacksdk`` is used. Since ``openstacksdk`` should be already present as a dependency to ``shade`` no additional actions are required
```

Comment:
period at end of statement.

Reply:
Agree it should have been there from the beginning. My item is not the only one without period, so adding it in other places too